src/hotspot/share/opto/loopnode.hpp

```diff
@@ -1630,6 +1630,51 @@ class PhaseIdealLoop : public PhaseTransform {
   // same block.  Split thru the Region.
   void do_split_if(Node *iff, RegionNode** new_false_region = nullptr, RegionNode** new_true_region = nullptr);
 
+private:
+  // Class to keep track of wins in split_thru_phi.
+  class SplitThruPhiWins {
+  private:
+    // Sum of all wins regardless of where they happen.
+    int _total_wins;
+    // Number of wins on a loop entry edge, which only pays dividens once per loop execution.
```

Comment:
You should specify that "If the split is through a loop head", otherwise `0`. Also, typo `dividends`